swaglyrics_backend/issue_maker.py

```diff
@@ -444,6 +444,6 @@ def hello():
     return render_template('hello.html', unsupported_songs=data)
 
 
-if __name__ == '__main__':
+if not token and gh_token != '':
```

Comment:
```suggestion
if not token and gh_token:
```
same difference, I'd also add a comment